sandbox/scripts/bootstrap.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Is this script just for debugging?

Reply:
It's used in the Dockerfile to start the server